modules/calib3d/test/test_solvepnp_ransac.cpp

```diff
@@ -837,6 +837,34 @@ TEST(Calib3d_SolvePnPRansac, double_support)
     EXPECT_LE(cvtest::norm(t, Mat_<double>(tF), NORM_INF), 1e-3);
 }
 
+
+TEST(Calib3d_SolvePnPRansac, bad_input_points)
+{
+    Mat pts2d = (Mat_<float>(6, 2) <<
```

Comment:
I would add some comments before this line to explain this test:

- with this specific data
- when computing the final pose using points in the consensus set with `SOLVEPNP_ITERATIVE` and `solvePnP()`
- an exception is thrown from `solvePnP` because there are 5 non-coplanar 3D points and the DLT algorithm needs at least 6 non-coplanar 3D points
- with PR #19253 we choose to return `true`, with the pose estimated from the MSS stage instead of throwing the exception

Reply:
Thanks, it do need some explanation.